samples/fpga/gemm_systolic_vectorized.py

```diff
@@ -77,21 +81,19 @@ def make_copy_to_host_state(sdfg, vec_width=1):
     C_device = state.add_read("C_device")
     C_host = state.add_write("C")
 
-    state.add_memlet_path(C_device,
-                          C_host,
-                          memlet=dace.Memlet(
-                              "C[0:N, 0:M/{}]".format(vec_width)))
+    state.add_memlet_path(C_device, C_host, memlet=dace.Memlet("C[0:N, 0:M]"))
 
     return state
 
 
-def make_read_A(state):
+def make_read_A(sdfg, state, vtype):
 
     # A is stored with plain data type
     entry, exit = state.add_map("read_A", {
-        "n0": "0:N/P",
+        "n0": "0:N//TN",
```

Comment:
Could you explain somewhere the meaning of the different symbols (in particular TN, TM and W)?

Reply:
I've added a comment giving an overview of all symbols used in b2922290